### PR TITLE
circle_4oh: pass circle CI builds again

### PR DESCRIPTION
**Pull request by fenollp**
master: https://github.com/2600hz/kazoo/pull/3933
related issue: https://github.com/ninenines/erlang.mk/issues/709

### CODE REVIEW

File: make/deps.mk [hunk 1]
```diff
@@ -22,7 +22,6 @@ dep_detergent = git https://github.com/pap/detergent e86dfeded3e4f9f3f9278c6a1ae
 dep_jiffy = hex 0.14.7
 dep_nklib = git https://github.com/NetComposer/nklib
 dep_plists = git https://github.com/essen/plists
-dep_erlydtl = git https://github.com/erlydtl/erlydtl 0.12.0
```
Comment (lazedo) [inline make/deps.mk]: why are we removing this ?

File: Makefile [hunk 2]
```diff
@@ -55,7 +55,7 @@ clean-deps:
 	$(if $(wildcard deps/), rm -r deps/)
 
 .erlang.mk:
-	wget 'https://raw.githubusercontent.com/ninenines/erlang.mk/2017.05.18/erlang.mk' -O $(ROOT)/erlang.mk
+	wget 'https://raw.githubusercontent.com/ninenines/erlang.mk/2017.07.06/erlang.mk' -O $(ROOT)/erlang.mk
```
Comment (lazedo) [inline Makefile]: if erlang.mk team is deleting tags, maybe we should fork to ensure we use a valid tag over time ?

Comment (fenollp) [inline Makefile]: the tag deletion episode was a one-of so we can still rely on that IMO

File: make/deps.mk (hunk 1 above)
Comment (fenollp) [inline make/deps.mk]: cause it appears their master branch has less issues (today's merl issue is due to that)

**Pull request merged**